releases/plugins/nightly/settings

```diff
@@ -0,0 +1 @@
+FOREMAN_VERSION='nightly'
```

Comment:
I'd prefer a solution similar to `client` where in `releases/plugins/settings` we set `FOREMAN_VERSION=$VERSION` and be done with it.
```suggestion
```